src/getHumanAge.test.js

```diff
@@ -7,4 +7,39 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('From 0 to 14 cat or dog years give 1 human year', () => {
+    const foo = getHumanAge(0, 14);
```

Comment:
Consider using a more descriptive variable name instead of 'foo'. For example, you could use 'humanAges' 📝